src/HDF5Msg.chpl

```diff
@@ -2169,11 +2483,12 @@ module HDF5Msg {
         var valSubdoms: [fD] domain(1);
         var segSubdoms: [fD] domain(1);
         var skips = new set(string);
+        var vskips = new set(string);
         var len: int;
         var nSeg: int;
 
         (segSubdoms, nSeg, skips) = get_subdoms(filenames, dset + "/" + SEGMENTED_OFFSET_NAME, validFiles);
-        (valSubdoms, len, skips) = get_subdoms(filenames, dset + "/" + SEGMENTED_VALUE_NAME, validFiles);
+        (valSubdoms, len, vskips) = get_subdoms(filenames, "%s/%s/Limb_0".format(dset, SEGMENTED_VALUE_NAME), validFiles);
```

Comment:
should this have `Limb_0` i feel like that would only apply to bigint segarray and this gets run for all kinds of segarrays right? or am i missing something